src/app/modules/superNodeProgram/superNodeProgram.html

```diff
@@ -144,7 +144,7 @@ <h3>{{ 'ACCOUNT_NAME' | translate }}</h3>
                 <!-- Password Field -->
                 <password-input common="$ctrl.common"></password-input>
                 <!-- Enroll button is disabled if already pressed or no password -->
-                <button class="btn btn-success" type="submit" ng-disabled="$ctrl.okPressed || ($ctrl._Wallet.algo !== 'trezor' && $ctrl._Wallet.algo !== 'ledger') && !$ctrl.common.password.length || $ctrl.formData.enrollAddress.length !== 40" ng-click="$ctrl.sendEnroll()"
+                <button class="btn btn-success" type="submit" ng-disabled="$ctrl.okPressed || ($ctrl._Wallet.algo !== 'trezor' && $ctrl._Wallet.algo !== 'ledger') && !$ctrl.common.password.length || $ctrl.formData.enrollAddress.length !== 40 || !$ctrl.isValid()" ng-click="$ctrl.sendEnroll()"
```

Comment:
i think `$ctrl.formData.enrollAddress.length !== 40` is unneeded because it is checked in isValid